### ISSUE TITLE
error resolving dependency for graphql@15.4.0

### ISSUE THREAD

**Issue by cryptoB0T** [helfer/graphql-disable-introspection#2]
not able to install due to:
error resolving graphql@^15.4.0 

```
❯ npm install -save graphql-disable-introspection                                                                                                 [ruby-2.6.3p62]

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: my-app@0.1.0
npm ERR! Found: graphql@15.4.0
npm ERR! node_modules/graphql
npm ERR!   graphql@"^15.4.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer graphql@"^0.9.2 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0" from graphql-disable-introspection@1.1.0
npm ERR! node_modules/graphql-disable-introspection
npm ERR!   graphql-disable-introspection@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/cryptoB0T/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/cryptoB0T/.npm/_logs/2021-01-19T17_19_01_642Z-debug.log

**Comment by supermonkeybrainz** [comment]
Looks like the peer dependencies need to be updated.  Would very much appreciate!

**Comment by helfer** [comment]
Yup, let me see if I should add that version to peerDependencies, or if there's a better way to solve this kind of thing now. Seems kind of silly to have to update this list every time there's a new major version of graphql-js if it's still compatible.

**Comment by helfer** [comment]
Published 1.2.0, which should fix this. Let me know if it doesn't.
